src/test/spec/mod.rs

```diff
@@ -43,6 +45,12 @@ where
         let json: Value =
             serde_json::from_reader(File::open(test_file_full_path.as_path()).unwrap()).unwrap();
 
-        run_test_file(bson::from_bson(Bson::from(json)).unwrap()).await
+        run_test_file(
```

Comment:
I ran into some errors here due to our manual handling of extjson, so to help me trace them, I added logging for the exact file that we failed to parse in the case that it failed.